src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -106,6 +106,14 @@ Runs diagnostics (such as type-checking within `.astro` files) against your proj
 
 This command is intended to be used in CI workflows.
 
+<h3>Flags</h3>
+
+Use these flags to customize the behavior of the command.
+
+#### `--watch`
+
+The command will watch for any change to `.astro` files, and will report errors, if there are any.
```

Comment:
```suggestion
The command will watch for any changes to `.astro` files, and will report any errors.
``` 

If this is also happening "real time" or "as they arise/occur", a phrase like that to distinguish from the regular `check` command that just checks at the moment of the command, could be nice here.